sdcm/tester.py

```diff
@@ -474,6 +474,20 @@ def get_cluster_aws(self, loader_info, db_info, monitor_info):
                 self.fail('Unsupported parameter type: {}'.format(type(n_db_nodes)))
         if db_info['type'] is None:
             db_info['type'] = self.params.get('instance_type_db')
+        if db_info['disk_size'] is None:
+            db_info['disk_size'] = self.params.get('aws_root_disk_size_db', default=None)
+        if db_info['device_mappings'] is None:
+            if db_info['disk_size']:
+                db_info['device_mappings'] = [{
+                    "DeviceName": self.params.get("aws_root_disk_name_db", default="/dev/sda1"),
```

Comment:
you didn't added this option to the sct_config

Reply:
please add `aws_root_disk_name_db` to the `sct_config`

Reply:
You are right, I missed it.

Reply:
Fixed.